### DISCUSSION TITLE
absorb transactions are not processed

### DISCUSSION THREAD

**Discussion by tatikoma** [Chia-Network/chia-blockchain#9045] [Support]
Hello,

I am pool operator. Today i noticed that all absorb transactions are stopped.
Its happened not for my pool only, it also happened for some other pools.

I checked logs, and found that blocks filled for 35%, so there is space to include my absorb transaction, but it wasnt happened.
I was increased mempool size from 50 to 100, but nothing changed.

My pool trying to absorb reward for more than 3 hours without success.

Any ideas ?

**Comment by RnMss** [comment]
I'm having the same problem.
Should we add a fee for absorb transaction? Can Chia team provide some sample code?

**Reply by tatikoma** [reply]
https://github.com/Chia-Network/pool-reference/pull/258/commits/749a08e83311cea50c7f86542a12891f06d81976

This has been solved my issue. I hope that helps.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
